docs/dev/oauth_for_integrations.md

```diff
@@ -59,43 +59,72 @@ The client is the component of an application that enables users to authorize th
 
    For more information about implementing the OAuth protocol with Datadog, see [Datadog OAuth2][1]. For more information about building and publishing an integration, see the [Integrations developer documentation][5].
 
-6. Test the OAuth protocol by clicking **Test Authorization** on your client's details page. This directs you to the onboarding URL and starts the authorization flow that a customer takes. 
+### Test the OAuth client
 
-    Before your client is published, only members of your Datadog organization can authorize against your client during testing. 
+Once you have implemented the OAuth protocol, you should test your OAuth client to ensure that you can send data into Datadog, or pull data out, according to your use case. 
 
-7. To validate that OAuth is working properly, make a request to the `marketplace_create_api` endpoint with your token in the headers of the request. 
+**Note**: Until your OAuth client is published, you can only authorize the client from your sandbox organization. This means that you can only send data into or pull data out of your sandbox account until the client is published. 
```

Comment:
It's really until the tile is published because they __cannot__ authorize from other orgs with their published client right when they go through the publishing flow.

Reply:
```suggestion
**Note**: Until your integration tile is published, you can only authorize the OAuth client from your sandbox organization. This means that you can only send data into or pull data out of your sandbox account until the client is published. 
```

Reply:
Makes sense to me :) Thanks!

Reply:
Thanks all, committed this